src/modules/r.rs

```diff
@@ -0,0 +1,87 @@
+use regex::Regex;
+
+use super::{Context, Module, RootModuleConfig};
+
+use crate::configs::r::RConfig;
+use crate::formatter::StringFormatter;
+use crate::utils;
+
+const R_VERSION_PATTERN: &str = r" (?P<rversion>\d+\.\d+\.\d+) ";
+
+/// Creates a module with the current Node.js version
+///
+/// Will display the Node.js version if any of the following criteria are met:
+///     - Current directory contains a `.js` file
+///     - Current directory contains a `package.json` or `.node-version` file
+///     - Current directory contains a `node_modules` directory
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let is_r_project = context.try_begin_scan()?.set_extensions(&["R"]).is_match();
+    if !is_r_project {
+        return None;
+    }
+
+    let r_version = utils::exec_cmd("r", &["--version"])?.stderr;
```

Comment:
This most likely won't work on unix-likes because the binary is usually installed under the name `R`.

```
bash-5.0$ r --version
bash: r: command not found
```

Reply:
That is true, let me change this to an upper-case `R`.